md_api/src/main/java/com/wipro/ats/bdre/md/api/TermJob.java

```diff
@@ -41,6 +42,17 @@
             {"p", "process-id", "Process Id of the process to terminate"},
     };
 
+
+    public TermJob() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+    @Autowired
+    private JobDAO jobDAO;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)